modules/common/src/test/java/com/ibm/cloud/cloudant/internal/ValidationTest.java

```diff
@@ -14,16 +14,50 @@
 
 import java.util.Map;
 import java.util.Map.Entry;
+import java.util.concurrent.TimeUnit;
+
 import com.ibm.cloud.cloudant.common.SdkCommon;
+import com.ibm.cloud.sdk.core.http.HttpConfigOptions;
 import com.ibm.cloud.sdk.core.security.NoAuthAuthenticator;
+import okhttp3.OkHttpClient;
 import org.testng.annotations.Test;
+import static org.testng.Assert.assertEquals;
 import okhttp3.HttpUrl;
 import okhttp3.Request;
 
 public class ValidationTest {
     
     private final String dbName = "testDatabase";
     private final String docId = "_testDocument";
+
+    @Test
+    void validateDefaultReadTimeoutSetting() {
+        // Default 2.5 minutes read timeout is set by default.
+        CloudantBaseService cloudantBaseService = new CloudantBaseService(null, new NoAuthAuthenticator()) {
+        };
+        assertEquals(TimeUnit.MILLISECONDS.toSeconds(cloudantBaseService.getClient().readTimeoutMillis()), 150);
```

Comment:
Should we also check the default `callTimeoutMillis()` value?

Reply:
I can add such test if you insist, but that would test the timeout setting of core SDK. This change does not modify the default value of `callTimeout`.

Reply:
Ok we can keep this as-is.